ouroboros-consensus/src/Ouroboros/Consensus/Storage/ChainDB/Impl/ChainSel.hs

```diff
@@ -251,18 +252,19 @@ addBlockSync cdb@CDB {..} BlockToAdd { blockToAdd = b, .. } = do
         VolDB.putBlock cdbVolDB b
```

Comment:
This alternative currently delivers the `WrittenToDisk` promise after chain selection, whereas it could deliver it before. Is there a reason to favor one option over the other? "Promptness" of written-to-disk would favor changing this.

Reply:
You're totally right, I glossed over it. I'll fix it.